api-docs/openapi-diff.js

```diff
@@ -0,0 +1,38 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+/* eslint import/no-unresolved:0 */
+
+const fs = require('fs');
```

Comment:
Can we write this in typescript?

Reply:
Oh yea, one moment

Reply:
there is also `watch.js` and `index.js` that are written this way. I'd be in favor of taking the time to convert them, i just haven't had the time to get in the weeds of how we'd build/compile it as these are just scripts that are ran, but if it's simple to do i'd be in favor.

Reply:
looks like there's already `ts-node` included as a dep so hopefully shouldn't be too difficult.

I may also make a separate PR to convert the others to TS if i get this one to work first.

Reply:
sounds good, thanks @nhsiehgit !